content/Products/RHOBS/onboarding-a-tenant-into-rhobs.md

```diff
@@ -0,0 +1,84 @@
+3. What is the expected or forecasted growth in collected data? X% growth in data per month?
+4. How many users will require read access? X team members? The entire OpenShift org?
+5. How many clients will be emitting data? 1 Prometheus server?
+6. How long should data be retained? 2 weeks?
+7. How many independent service accounts do you need to write data into the platform?
+
+## 2. Create a Customer Portal Account
+
+Create a Red Hat customer portal account to group all users from your team who should have access to the data. Note: any member of your account will be able to read the data collected by Observatorium for your tenant. https://sso.redhat.com
+
+## 3. Create Service Accounts for Reading/Writing Data
+
+All requests reading and writing data into the Observatorium platform must be authenticated and authorized. For this reason, each tenant must provision service accounts for their data writers, e.g. Prometheus, Promtail, etc, and their data readers, e.g. Grafana to be able to make requests. Services that would also like to test their configuration in a staging environment should create additional service accounts for the staging Observatorium instance. Services must have at least one service account, which they may choose to reuse among all of their data clients. If services would like to authenticate each client independently, then an additional service account is needed for each client. These service accounts, both for production and staging, must be created in the production external SSO environment, that is, sso.redhat.com. Service accounts may be requested via a ticket for the IT User team in Service Now at the URL help.redhat.com. The service account request should specify the following requirements:
+
+1. define two new service accounts in the production sso.redhat.com: observatorium-<tenant>, observatorium-<tenant>-staging;
+2. enable the clients to use the client credentials grant flow;
+3. add observatorium-telemeter to the allowed audience field of the observatorium-<tenant> account;
+4. add observatorium-telemeter-staging to the allowed audience field of the observatorium-<tenant>-staging account;
+5. set the email addresses of the accounts to: <your-email>@redhat.com;
+6. set the preferred_username of the observatorium-<tenant> account to: service-account-observatorium-<tenant>; and
+7. set the preferred_username of the observatorium-<tenant>-staging account to: service-account-observatorium-<tenant>-staging.
+8. append one new allowed callback URL to the list of allowed callback URLs for the observatorium-telemeter service account: https://observatorium.api.openshift.com/oidc/<tenant>/callback
+9. append one new allowed callback URL to the list of allowed callback URLs for the observatorium-telemeter-staging service account: https://observatorium.api.stage.openshift.com/oidc/<tenant>/callback
+
+## 4. Authorize Service Accounts
```

Comment:
I think what I am missing here is any mention about the infra tenant needs to send data to RHOBS, added ticket for this https://github.com/rhobs/handbook/issues/26